components/tab/theme.css

```diff
@@ -0,0 +1,66 @@
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-animations';
+
+.title-tab {
+  color: var(--color-neutral-darkest);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  outline: 0;
+  user-select: none;
+  position: relative;
+  text-decoration: none;
+  background-color: var(--color-neutral-lightest);
+  border: none;
+  border-radius: 0;
+
+  &:hover {
+    background-color: color(var(--color-neutral-darkest) a(24%));
+    cursor: pointer;
+  }
+
+  &:focus {
+    background-color: color(var(--color-neutral-darkest) a(12%));
+    box-shadow: 0 0 0 2px color(var(--color-neutral-darkest) a(24%));
+  }
+
+  &::after {
+    content: '';
+    position: absolute;
+    display: block;
+    color: var(--color-teal-darkest);
+    box-shadow: inset 0px -3px 0px 0px var(--color-mint);
+    height: 3px;
+    width: 100%;
+    bottom: 0;
+    transition: transform var(--animation-duration) var(--animation-curve-fast-out-slow-in);
```

Comment:
didn't you say it was too slow?

 :ok: by @lowiebenoot on 30/11/2017 14:59:57

Reply:
I talked to Stephen, he said I could take this and he would talk with you about changing the variable.

 :ok: by @InstaK on 30/11/2017 18:33:15